spec/models/activity/import/field_spec.rb

```diff
@@ -3,8 +3,23 @@
     subject { described_class.all }
 
     it "returns an array of all Field objects" do
-      expect(subject.count).to eq 50
+      expect(subject.count).to eq 49
       expect(subject).to all(be_a(described_class))
+
+      linked_activity_id_field = subject.select { |field| field.attribute_name == :linked_activity_id }
+      expect(linked_activity_id_field).to be_empty
```

Comment:
`be_empty` or `be_nil`?

Reply:
Select returns an empty array when not found, there may be another tidier way of doing this though

Reply:
Ah true. Given you're just looking for one field, would be make more sense to use `.find` here and then assert on `nil` (in both tests)?